recipes/Osi/meta.yaml

```
@@ -0,0 +1,67 @@
+{% set name = "coin-or-osi" %}
+{% set version = "0.108.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/coin-or/Osi/archive/releases/{{ version }}.tar.gz
+    sha256: 984a5886825e2da9bf44d8a665f4b92812f0700e451c12baf9883eaa2315fad5
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pkg-config
+    - make  # [unix]
+  host:
+    - coin-or-utils
+    - zlib
+    - bzip2
+    - libblas
+    - libcblas
+    - liblapack
+  run_constrained:
+    - coincbc * metapackage
```

Where is this coming from?

---

It's a nonexistent build of coincbc to prevent parallel installation.

---

Could we make it
```suggestion
    # prevent parallel installation of coincbc
    - coincbc <0a0
```
then? The `<0a0` is a pattern I'd recognize as "prevent installation" ;).
(Also adding a descriptive comment would be good. Best would be to also give a reason why this is done, unlike in the suggestion above.)

---

no, that's not what I want. I want to depend on a non-existant metapackage. 
this metapackage _will_ be created once these new coin-or libs are merged, and then become available.

---

I'm not a fan of completely overriding the build strings because of "losing" the build number.
I'd at least go for
```suggestion
    - coincbc * *_metapackage
```

---

done! good suggestion